src/mobility/components/CarImage.tsx

```diff
@@ -1,35 +1,63 @@
-import {Image, View} from 'react-native';
+import {Image, ImageStyle, View} from 'react-native';
 import React from 'react';
 import {ThemeIcon} from '@atb/components/theme-icon';
-import {Car} from '@atb/assets/svg/mono-icons/transportation';
+import {Car} from '@atb/assets/svg/mono-icons/transportation-entur';
+import {StyleSheet, useTheme} from '@atb/theme';
+import {ThemeText} from '@atb/components/text';
 
 type Props = {
-  uri: string | undefined;
+  uri?: string | undefined;
+  plus?: number | undefined;
 };
-export const CarImage = ({uri}: Props) => {
+
+export const CarImage = ({uri, plus}: Props) => {
+  const style = useSheetStyle();
+
   return uri ? (
     <Image
-      style={{height: 75, width: 100, resizeMode: 'contain'}}
+      style={[style.container as ImageStyle, style.image as ImageStyle]}
       source={{uri}}
     />
+  ) : plus ? (
+    <PlusContainer plus={plus} />
   ) : (
     <DefaultCarImage />
   );
 };
 
 const DefaultCarImage = () => {
+  const {theme} = useTheme();
+  const style = useSheetStyle();
+  return (
+    <View style={[style.container, style.defaultImage]}>
+      <ThemeIcon size="large" fill={theme.text.colors.secondary} svg={Car} />
+    </View>
+  );
+};
+
+const PlusContainer = ({plus}: Props) => {
+  const style = useSheetStyle();
   return (
-    <View
-      style={{
-        height: 75,
-        width: 100,
-        backgroundColor: '#d8d8d8',
-        borderRadius: 5,
-        justifyContent: 'center',
-        alignItems: 'center',
-      }}
-    >
-      <ThemeIcon size="large" fill="#A9AEB1" svg={Car} />
+    <View style={[style.container, style.defaultImage]}>
+      <ThemeText style={style.defaultImage}>+{plus}</ThemeText>
     </View>
   );
 };
+
+const useSheetStyle = StyleSheet.createThemeHook(() => {
+  return {
+    container: {
+      height: 40,
+      width: 40,
+      borderRadius: 8,
+      backgroundColor: '#d8d8d8',
+    },
+    image: {
+      resizeMode: 'cover',
```

Comment:
In the Figma sketches, 'contain' is used. However as far as I can see, this only applies to Otto images which are transparent, while Hyre images are not transparent and look better with 'cover'. Potentially worth checking whether the image is transparent to determine which one to use. Thoughts on this @oyvindmikkelsenatb @KristianSelnas? (see car image in the bottom sheet)


| Cover | Contain |
|--------|-------|
| <img width="200" src="https://github.com/AtB-AS/mittatb-app/assets/134292729/f8df3e53-d44c-44fa-8239-e247f3df7e38" /> | <img width="200" src="https://github.com/AtB-AS/mittatb-app/assets/134292729/53b14e0a-48e6-4fb9-a894-dd4e738909fd" /> |
| <img width="200" src="https://github.com/AtB-AS/mittatb-app/assets/134292729/41058033-852d-4f72-8dcc-b05c073ea511" /> | <img width="200" src="https://github.com/AtB-AS/mittatb-app/assets/134292729/798a850b-55c2-45db-ac45-c1f1d7ab08db" /> |

Reply:
I agree that cover looks better for non-transparent, rectangular images.  I'm not sure how much work detecting opacity and handling that is, but if it's non-trivial I vote for 'cover'

Reply:
No checking of specific operators allowed in the app code at least 🚫😅 If we want anything like that, it needs to be in the modelling in Firestore. I think just selecting a `resizeMode` is better.

Reply:
I think having a `cover` is acceptable for now, at least if the user wants to check a more detailed image, they can just open the operator App instead of looking at the preview. It's just there as an indicator. At least by using `cover`, the one shown in Otto operator is better than using `contain` on Hyre operator.